packages/manager/src/utilities/pricing/dynamicPricing.ts

```
@@ -41,7 +41,7 @@ export const getDCSpecificPrice = ({
   flags,
   regionId,
 }: DataCenterPricingOptions) => {
-  if (!flags?.dcSpecificPricing) {
+  if (!flags?.dcSpecificPricing || !regionId) {
     return basePrice.toFixed(2);
   }
```

Good idea to fix this. This will however fail silently and we would provide the wrong price, which could be bad. That's not ideal but i dunno if there's a better way.. perhaps we can discuss this further

---

Hmm. I see what you mean. I'll have to do some thinking 💭 on how to handle that